MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,429 @@
+        contentView.addSubview(caracter1label)
+        
+        caracter2label = UILabel()
+        caracter2label.text = "Characters"
+        contentView.addSubview(caracter2label)
+        
+        directorlabel = UILabel()
+        directorlabel.text = "Director"
+        contentView.addSubview(directorlabel)
+        
+        screenplay1label = UILabel()
+        screenplay1label.text = "Screenplay"
+        contentView.addSubview(screenplay1label)
+        
+        screenplay2label = UILabel()
+        screenplay2label.text = "Screenplay"
+        contentView.addSubview(screenplay2label)
+        
+        screenplay3label = UILabel()
+        screenplay3label.text = "Screenplay"
+        contentView.addSubview(screenplay3label)
+    }
+    
+    func styleViews() {
+        posterView.backgroundColor = .blue
+        
+        posterImage.contentMode = .scaleAspectFill
+        posterImage.clipsToBounds = true
+        
+        titleLabel.font = UIFont.boldSystemFont(ofSize: 26)
+        durationLabel.font = UIFont.boldSystemFont(ofSize: 16)
+        
+        contentView.backgroundColor = .white
+        
+        overviewLabel.font = UIFont.boldSystemFont(ofSize: 22)
+        
+        
+        caracter1.font = UIFont.boldSystemFont(ofSize: 14)
+        caracter2.font = UIFont.boldSystemFont(ofSize: 14)
+        director.font = UIFont.boldSystemFont(ofSize: 14)
+        screenplay1.font = UIFont.boldSystemFont(ofSize: 14)
+        screenplay2.font = UIFont.boldSystemFont(ofSize: 14)
+        screenplay3.font = UIFont.boldSystemFont(ofSize: 14)
+        
+        caracter1label.font = UIFont.systemFont(ofSize: 14)
+        caracter2label.font = UIFont.systemFont(ofSize: 14)
+        directorlabel.font = UIFont.systemFont(ofSize: 14)
+        screenplay1label.font = UIFont.systemFont(ofSize: 14)
+        screenplay2label.font = UIFont.systemFont(ofSize: 14)
+        screenplay3label.font = UIFont.systemFont(ofSize: 14)
+    }
+    
+    func addConstraints() {
+        let bounds = UIScreen.main.bounds
+        safeAreaView.snp.makeConstraints {
+            $0.edges.equalTo(view.safeAreaLayoutGuide)
+        }
+        
+        posterView.snp.makeConstraints{
+            $0.leading.trailing.top.equalToSuperview()
+            $0.bottom.equalToSuperview().inset(bounds.height*0.5)
+        }
+        
+        posterImage.snp.makeConstraints{
+            $0.edges.equalToSuperview()
+        }
+        
+        scrollView.snp.makeConstraints {
+            $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
+            $0.top.equalTo(posterView.snp.bottom)
+        }
+        
+        contentView.snp.makeConstraints {
+            $0.edges.equalTo(scrollView)
+            $0.width.equalTo(view)
+        }
+        
+        addConstraintsInPosterView()
+        addConstraintsInScroolView()
+    }
+    
+    func addConstraintsInPosterView() {
+        popularityLabel.snp.makeConstraints{
```

Comment:
You have a general problem with constraining labels - you do not add a trailing constraint. That means that it will not know its trailing boundary and will never truncate or go into a new line. Extra text will just disappear.